src/components/DimensionComparator.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { Card, Slider, Typography, Tabs } from "antd";
+import "../styles/Dimension.css";
+import "../styles/CompareCharts.css";
+import moment from "moment";
+
+import { Dimension as DimensionType } from "../types/dimension";
+const { TabPane } = Tabs;
+
+interface DimensionComparatorProps {
+  dimensions: DimensionType[];
+  dates: Date[];
+  isPreview: boolean;
+}
+
+interface labeledMarks {
+  [key: number]: any;
```

Comment:
I think we already have a type in types/SliderMarks, but this will change in Eric's PR, still probs best to use that typ though

Reply:
It is not suitable for this implementation as I am adding an extra key/value pair on line 73 to this object to keep track of dates. I am not sure how to extract this data from the JSX in the `label` property :(

Reply:
ahh okay, dw about it then